src/utils/index.js

```diff
@@ -1,20 +1,32 @@
 const fetch = typeof window !== 'undefined' ? window.fetch : require('node-fetch')
 const Multihash = require('multihashes')
 const sha256 = require('js-sha256').sha256
+const sigUtil = require('eth-sig-util')
 
 const HTTPError = (status, message) => {
   const e = new Error(message)
   e.statusCode = status
   return e
 }
 
-const getMessageConsent = (did) => (
-  'Create a new 3Box profile' + '\n\n' + '- \n' + 'Your unique profile ID is ' + did
-)
+const getMessageConsent = (did, timestamp) => {
+  let msg = 'Create a new 3Box profile' + '\n\n' + '- \n' + 'Your unique profile ID is ' + did
+  if (timestamp) msg += ' \n' + 'Timestamp: ' + timestamp
+  return msg
+}
 
 module.exports = {
   getMessageConsent,
 
+  recoverPersonalSign: (msg, personalSig) => {
+    if (!msg || !personalSig) throw new Error('recoverPersonalSign: missing arguments, msg and/or personalSig')
+    const msgParams = {
+      data: msg,
+      sig: personalSig
+    }
+    return sigUtil.recoverPersonalSignature(msgParams)
```

Comment:
Would it be a lot of additional work to recover the signature with ethersjs? 
https://docs.ethers.io/ethers.js/html/api-utils.html?highlight=signature#elliptic-curve

Reply:
I guess we just need to prepend the right data for an eth-signature?

Reply:
Since we're already importing ethers anyway it would remove the dependency on eth-sig-util

Reply:
yeah good idea, switched over